draft-trammell-quic-spin.md

```diff
@@ -98,9 +98,57 @@ protocol.
 
 # The Spin Bit Mechanism {#mechanism}
 
-\[EDITOR'S NOTE: Marcus to write. Modify slightly from PR609 design, spin only
-on short header packets. Coordinate with work on reserved bits in the short
-header in the transport draft?]
+The latency spin bit enables latency monitoring from observation points on
+the network path. The bit is set by the endpoints in the following way:
+
+* The server sets the spin bit value to the value of the
+  spin bit in the packet received from the client with
+  the largest sequence number.
+
+* The client sets the spin bit value to the opposite
+  of the value set in the packet received from the server with the
+  largest sequence number, or to 0 if no packet as been received yet.
+
+Observation points can estimate the network latency by monitoring these
+changes in the latency spin bit.
+If packets are delivered in sequence, this procedure will cause the spin
+bit to change value in each direction once per round trip.
+
+Out of sequence packets can cause spurious spins, it is therefore recommended
+that monitoring agents keep track of packet sequence number and / or use 
+filtering.
+
+The latency spin bit handling is similar to the
+"Alternate Marking method for passive performance monitoring"
+described in [I-D.draft-ietf-ippm-alt-mark].
```

Comment:
I'm wondering if we should leave the alt-mark mention out here, and move it to the #using section?

Reply:
Makes sense